test/unit_tests/products/stations/test_igra.py

```diff
@@ -0,0 +1,40 @@
+"""
+Tests for the pansat.products.reanalysis.ncep module.
+"""
+
+from datetime import datetime
+import os
+import pytest
+import pansat.products.stations.igra as igra
+import random
+
```

Comment:
Would be good to test the other member functions of the IGRASoundings

Reply:
- I added two tests: one for the download based on variable instead of station and one to unzip and open the downloaded file. 
- I removed the function filename_to_date since the filenames do not contain the dates. 
- The remaining member functions are called in the download or init function (e.g. get_metadata, get_filename, etc)